app/Http/APIV1/Flower/Controllers/FlowerController.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace App\Http\APIV1\Flower\Controllers;
+
+use App\Http\APIV1\Flower\Resources\FlowerResource;
+use App\Http\APIV1\Flower\Resources\FlowerCollection;
+use App\Http\APIV1\Room\Resources\RoomResource;
+use App\Http\APIV1\Flower\Requests\AddFlowerRequest;
+use App\Http\APIV1\Flower\Requests\UpdateFlowerFieldsRequest;
+use App\Http\APIV1\Flower\Requests\UpdateFlowerRequest;
+use App\Http\APIV1\Flower\Requests\GetFlowersRequest;
+use App\Domain\Flowers\Actions;
+use App\Domain\Flowers\Actions\GetFlowers;
+use App\Domain\Flowers\Actions\UpdateFlower;
+use App\Http\APIV1\Common\EmptyJsonResource;
+use App\Http\APIV1\Common\ErrorJsonResource;
+use App\Http\Controllers\Controller;
+use Error;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Validation\ValidationException;
+use PhpParser\Node\Expr\Empty_;
+
+class FlowerController extends Controller
+{
+    public function create(AddFlowerRequest $request)
+    {
+        $result = [];
+        $return = [];
+        $action = new Actions\AddFlower();
+        try {
+            $result['data'] = $action->execute($request->validated());
+            $return = (new FlowerResource($result['data']))->response()->setStatusCode(201);
```

Comment:
Нужно прямо ресурс возвращать из контроллера https://laravel.com/docs/9.x/eloquent-resources#resource-responses
```suggestion
           return FlowerResource::make($action->execute($request->validated()));
```